student/src/test/resources/test-settings.xml

```diff
@@ -7,5 +7,9 @@
 	<entry key="ItembankDBName">itembank</entry>
 	<entry key="TDSArchiveDBName">archive</entry>
 	<entry key="TDSConfigsDBName">configs</entry>
-		
+
+	<entry key="EncryptionKey">Thisisanincrediblylongkeythatiscertainlylongerthantwentyfourcharacters</entry>
+	<entry key="datasource.jdbcUrl">jdbc:mysql://[URL]/session</entry>
+	<entry key="datasource.username">remoteuser</entry>
+	<entry key="datasource.password">[PASSWORD]</entry>

```

Comment:
The current setup is to have the DB credentials in the code which is obviously not good.  So I omitted them for the PR but would like to follow what you all are doing in order to run the tests on TeamCity cleanly.  Figured I could get some details on that and follow that pattern if possible.